solr/core/src/test/org/apache/solr/response/transform/TestSubQueryTransformer.java

```diff
@@ -127,7 +127,7 @@ public void testJohnOrNancySingleField() throws Exception {
      //System.out.println("p "+peopleMultiplier+" d "+deptMultiplier);
     assertQ("subq1.fl is limited to single field",
         req("q","name_s:(john nancy)", "indent","true",
-            "fl","name_s_dv,depts:[subquery]", 
+            "fl","dept_ss_dv,name_s_dv,depts:[subquery]", 
```

Comment:
Many tests in this class seem to have just been fortunate that `SolrIndexSearcher` ignored `fl` and retrieved all fields when not using lazy loading.